### ISSUE TITLE
Cobalt (II) chloride: Incorrect response at "max concentration"

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#167]
@twant, I just happen to notice this incorrect response for Cobalt (II) chloride. This issue should have been fixed in #154. Is it clear to you why the correct alert is not coming through here? 

When leaving the saturated zone for Cobalt (II) chloride, I got:
-  "No longer saturated. Solution lighter, max concentration."
![Screen Shot 2019-11-22 at 3 41 17 PM](https://user-images.githubusercontent.com/1902116/69453715-1682e800-0d3f-11ea-8ffb-765c761056d6.png)
 
**I think the correct response should be:**
- "At max concentration. Solution darkest."

Here's an example from Gold (III) chloride.
![Screen Shot 2019-11-22 at 3 43 19 PM](https://user-images.githubusercontent.com/1902116/69453834-5cd84700-0d3f-11ea-82f4-51ecdee81f41.png)

**Comment by terracoda** [comment]
I was able to repeat the issue with Solution Volume, too.
I didn't find this on a quick check on all the other solutes. It might just be an issue for this one.
![Screen Shot 2019-11-22 at 4 34 41 PM](https://user-images.githubusercontent.com/1902116/69456759-0a4e5900-0d46-11ea-98c4-e632ce1dd53e.png)

**Comment by twant** [comment]
@terracoda thanks for catching this! Can you send along the numerical values at which this is happening? I can't seem to replicate, but would love to get to the bottom of this!

**Comment by twant** [comment]
And is this from master, or from a dev version?

**Comment by terracoda** [comment]
It should be master than I checked, but I will check again, and grab values for you, too.

**Comment by terracoda** [comment]
@twant, I can't reproduce it either. Maybe I hadn't properly emptied my caches! Cobalt (II) chloride all looks good!
**Solution Values not checked**
![Screen Shot 2019-11-23 at 12 53 04 PM](https://user-images.githubusercontent.com/1902116/69482104-d7619f00-0df1-11ea-87a0-64e0bb22aaf4.png)
![Screen Shot 2019-11-23 at 1 02 54 PM](https://user-images.githubusercontent.com/1902116/69482135-260f3900-0df2-11ea-804d-72a19a5b1611.png)

**Solution Values checked**
![Screen Shot 2019-11-23 at 12 53 04 PM](https://user-images.githubusercontent.com/1902116/69482138-2e677400-0df2-11ea-8410-766a1b4cccc6.png)
![Screen Shot 2019-11-23 at 1 03 22 PM](https://user-images.githubusercontent.com/1902116/69482144-358e8200-0df2-11ea-8356-6c7f867ed7f0.png)

**Comment by terracoda** [comment]
Closing! Thanks for taking a second look @twant. I think it's all good! I'll be more careful about emptying my caches next time.

This sim is ready for QA in my opinion.

**Comment by terracoda** [comment]
Or whatever the next steps are :-) Awesome work!
